week-1/J-functions/exercise2.js

```diff
@@ -1,7 +1,32 @@
-function triple(number) {
-  // complete function here
+function halve(number) {
+  return number / 2;
 }
 
-var result = triple(12);
-
+var result = halve(12);
 console.log(result);
+
+function triple(result) {
+  return result * 3;
+}
+var results = triple(8);
+console.log(results);
```

Comment:
This is good. Just so you know, you can also log the result of the function to console directly. E.g. console.log(triple(8));